core/drivers/crypto/stm32/cipher.c

```diff
@@ -225,12 +197,21 @@ static TEE_Result stm32_cryp_cipher_allocate(void **ctx, uint32_t algo)
 	}
 }
 
-/*
- * Allocate the SW cipher data context for SAES peripheral.
- *
```

Comment:
why did you remove the documentation ?

Reply:
i think it is not useful. These are a local function and then label is already quite explicit.

Reply:
ok, valid point